src/com/couchbase/tools/performer/BuildDockerAnalyticsPythonPerformer.groovy

```diff
@@ -41,6 +41,10 @@ class BuildDockerAnalyticsPythonPerformer {
 
             if (build instanceof HasVersion) {
                 dockerBuildArgs.put("BUILD_FROM_VERSION", build.version())
+            } else if (build instanceof BuildMain) {
+                dockerBuildArgs.put("BUILD_FROM_REPO", 'MAIN')
+            } else if (build instanceof HasSha) {
+                dockerBuildArgs.put("BUILD_FROM_REPO", build.sha())
```

Comment:
If we standardised these flags we could probably do away with most per-performer logic for EA performers.  

(And technically we wouldn't even need jenkins-sdk for building them, since the Docker logic is now the full abstraction - but probably nice to keep a one-stop-shop for building any performer, and we may need it in future)